pkg/controller/apiserver/handlers/endpoint/handler.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package endpoint
+
+import (
+	"encoding/json"
```

Comment:
take a look at other files to see how we group imports, that will make the review process more efficient when we submit a PR against upstream antrea eventually

Reply:
Will make sure everything is consistent tomorrow morning.